src/send/SendSelectRecipientSearchInput.tsx

```diff
@@ -45,13 +45,11 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   search: {
-    height: 44,
-    alignItems: 'flex-start',
+    alignItems: 'center',
```

Comment:
Looking at the screenshots, seems like this is still not center aligned on iOS? Also cc @jophish who I think is fixing the alignment on a different PR